src/wasm/encoder.cc

```diff
@@ -90,6 +90,13 @@ void WasmFunctionBuilder::AddBody(
   }
 }
 
+void WasmFunctionBuilder::AddBody(const byte instruction, bool is_local) {
```

Comment:
I want to keep the encoder agnostic to actual wasm op codes. Those are supposed to be encapsulated in the walker. This function is just a special case that can easily take a single byte instead of an array of bytes.